core/src/main/java/com/radixdlt/consensus/liveness/PacemakerTimeoutCalculatorConfig.java

```diff
@@ -62,22 +62,36 @@
  * permissions under this License.
  */
 
-package com.radixdlt.consensus.bft;
+package com.radixdlt.consensus.liveness;
 
-import static java.lang.annotation.ElementType.FIELD;
-import static java.lang.annotation.ElementType.METHOD;
-import static java.lang.annotation.ElementType.PARAMETER;
-import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import com.google.common.base.Preconditions;
 
-import java.lang.annotation.Retention;
-import java.lang.annotation.Target;
-import javax.inject.Qualifier;
+public record PacemakerTimeoutCalculatorConfig(
+    long baseTimeoutMs,
+    double rate,
```

Comment:
IMO we should rename these to e.g. `consecutiveTimeoutSlowdownRate`, `consecutiveTimeoutMaxExponent` to make clear they just apply to that.